gnat2goto/ireps/ireps_generator.py

```diff
@@ -28,22 +28,37 @@
 
 import os
 import os.path
-import sys
 import json
 import argparse
 
 from pprint import pprint
 from glob import glob
 from copy import copy
 
+
+# Helper functions
+
 irep_type_to_ada_type = {
-   "irep"      : "Irep",
-   "bool"      : "Boolean",
-   "integer"   : "Integer",
-   "string"    : "String",
-   "gnat:sloc" : "Source_Ptr"
+    "irep"      : "Irep",
+    "bool"      : "Boolean",
+    "integer"   : "Integer",
+    "string"    : "String",
+    "gnat:sloc" : "Source_Ptr"
 }
 
+special_names = {"+"      : "op_add",
```

Comment:
odd indentation here

Reply:
Ah well, hasty copy paste. It's already fixed in my code. I will push it with the next changes, if you don't mind.

Reply:
@apaschos No, just commit the change of the fix in indentation, rebase (squash) the two commits together, then just force push and it will be fixed here.

If you fix it later, you will just be introducing irrelevant changes to another PR, and they also won't be logically coherent.